financial-services-accelerator/components/org.wso2.financial.services.accelerator.identity.extensions/src/main/java/org/wso2/financial/services/accelerator/identity/extensions/auth/extensions/response/handler/FSCodeResponseTypeHandlerExtension.java

```diff
@@ -93,6 +93,9 @@ public OAuth2AuthorizeRespDTO issue(OAuthAuthzReqMessageContext oauthAuthzMsgCtx
         } catch (FinancialServicesException e) {
             log.error("Error while invoking external service extension", e);
             throw new IdentityOAuth2Exception("Error while invoking external service extension");
+        } catch (JsonProcessingException e) {
+            log.error("Error while processing json", e);
```

Comment:
"Error while processing json" can be a constant since its used at multiple places

Reply:
Fixed with 7a2572989fb3d9d679eb9b9f1df2e5119a452ed5